### ISSUE TITLE
Skript not starting

### ISSUE THREAD

**Issue by Adrihun** [bensku/Skript#668]
I have downloaded the latest version of Skript **29**
i got this error when i started the server

https://pastebin.com/raw/S4uKLbsQ

**Is it because i'm using 1.8.9?**

**Comment by TheBentoBox** [comment]
Yes, Skript is no longer compatible with versions older than 1.9 as of dev28. I'd recommend downgrading to dev25 (or updating your server).

**Comment by fkmeloon** [comment]
Wait what! :(
@TheBentoBox That is horrible. Why would @bensku Do that...

Like 60% of people use **1.8.9**, and all of the big servers like Hypixel use **1.8-1.12**
I have been waiting for the **new update** that fixes the Java thing (so Scripts start faster)
and now @bensku doesn't support **1.8.9** anymore.

**Comment by Snow-Pyon** [comment]
@fkmeloon it never has been supported, it just magically worked (with some fixes that Bensku provided sometimes such as these were simple things to fix). You can still use the dev25.

**Comment by TheBentoBox** [comment]
No, 60% of people do not use 1.8.9, and he did this because 1.8 came out almost three years ago and continuing to support it became a burden. If you expect plugins to support you forever then you're pretty entitled. The longer you go without updating, the more often this will happen to you. Just get around to updating your server sometime and you won't keep losing out on plugins :v)

Anyways, this is the 2nd or 3rd time in as many days I've gotten into this conversation, so I'll be locking this one. This isn't really open for debate and as Pyon said, 1.8 was never officially supported. You're lucky it worked until now.

**Comment by bensku** [comment]
I needed to fix particles on 1/9 and sadly Spigot 1.8 doesn't support said particle API. Thus, it is *hard* to make it work on 1.8, at least without disabling particles completely. I would be happy to receive pull request that restores 1.8 compatibility - note that it would require good Java skills, as this is complex issue.

The fact that many people use 1.8 doesn't change anything - I have been clear that it has never been supported. If you want it to become supported, someone else needs to come and help me maintain it. That, or pay me for time I use with it.

Tl;dr update your server
